webapp/src/app.tsx

```diff
@@ -3951,8 +3951,7 @@ document.addEventListener("DOMContentLoaded", () => {
                 live,
                 force)
                 .then(() => {
-                    if (pxt.Util.isLocaleEnabled(useLang)) {
-                        lang.setCookieLang(useLang);
+                    if (pxt.BrowserUtils.setCookieLang(useLang)) {
```

Comment:
This is incorrect. We need to check isLocaleEnabled which checks appTheme.availableLocales before enabling the locale for a particular editor

Reply:
the check happens in setCookieLang

Reply:
That is not the same check. It checks all the languages. Each editor has a specific language checked.